### ISSUE TITLE
A list of non-functioning links on an iPhone 16 pro, iOS 18.1.1, US-EN language

### ISSUE THREAD

**Issue by ahrendsen** [FifiTheBulldog/ios-settings-urls#82]
Compiled with the assistance of @atnbueno ’s tool, thanks!

Title: "Settings ⭢ Accessibility ⭢ Display & Text Size ⭢ Larger Text" doesn't work
URL: prefs:root=ACCESSIBILITY&path=DISPLAY_AND_TEXT/Larger%20Text
Device: iPhone
OS version: 18.1.1
Language: en

**Comment by ahrendsen** [comment]
Title: "Settings ⭢ Accessibility ⭢ Apple TV Remote" doesn't work
URL: prefs:root=ACCESSIBILITY&path=Apple%20TV%20Remote
Devi
OS version: 18.1.1
Language: en

**Comment by ahrendsen** [comment]
Accidental close.

**Comment by xDraker** [comment]
Title: "Settings ⭢ Privacy and Security ⭢ Localiation" stopped working
URL: prefs:root=Privacy&path=LOCATION
Device: iPhone 13
OS version: 18.2
Language: es

**Comment by yanncabral** [comment]
Title: "Settings ⭢ Wi-Fi" stopped working
URL: Wi-Fi: prefs:root=WIFI
Device: iPhone 13
OS version: 18.2
Language: en

**Comment by ahrendsen** [comment]
Title: "Settings ⭢ Mail ⭢ Blocked" doesn't work
URL: prefs:root=MAIL&path=Blocked
Device: iPhone
OS version: 18.1.1
Language: en

**Comment by ahrendsen** [comment]
Title: "Settings ⭢ Privacy ⭢ Analytics & Improvements" doesn't work
URL: prefs:root=Privacy&path=PROBLEM_REPORTING
Device: iPhone
OS version: 18.1.1
Language: en

The link to privacy does work, but it doesn’t get through to “Analytics & Improvements”

UPDATE: In OS Version 18.3.1, it now does work.

**Comment by ahrendsen** [comment]
I realize now that putting multiple links for one device makes it more difficult to track the status of the individual links. For example, the link to Analytics & Improvements now does work in 18.3.1. In the future, I'll make an issue for each link.

**Comment by llvllo** [comment]
> Compiled with the assistance of [@atnbueno](https://github.com/atnbueno) ’s tool, thanks!
> 
> Title: "Settings ⭢ Accessibility ⭢ Display & Text Size ⭢ Larger Text" doesn't work URL: prefs:root=ACCESSIBILITY&path=DISPLAY_AND_TEXT/Larger%20Text Device: iPhone OS version: 18.1.1 Language: en

Updated URL: prefs:root=ACCESSIBILITY&path=DISPLAY_AND_TEXT/LARGER_TEXT#LARGER_TEXT
Tested on iPadOS 18.4.1 & iOS 18.5 Beta 2

**Comment by llvllo** [comment]
> Title: "Settings ⭢ Mail ⭢ Blocked" doesn't work URL: prefs:root=MAIL&path=Blocked Device: iPhone OS version: 18.1.1 Language: en
> 
> As first identified in [#81](https://github.com/FifiTheBulldog/ios-settings-urls/issues/

Updated URL in #89, Tested on iPadOS 18.4.1 & iOS 18.5 Beta 2

**Comment by llvllo** [comment]
> Title: "Settings ⭢ Accessibility ⭢ Apple TV Remote" doesn't work URL: prefs:root=ACCESSIBILITY&path=Apple%20TV%20Remote Devi OS version: 18.1.1 Language: en
> 
> Title: "Settings ⭢ Accessibility ⭢ Apple TV Remote" doesn't work URL: prefs:root=ACCESSIBILITY&path=Apple%20TV%20Remote Device: iPhone OS version: 18.1.1 Language: en

Updated URL: prefs:root=ACCESSIBILITY&path=APPLE_TV_REMOTE#AppleTVLiveTVButtons
Tested on iPadOS 18.4.1 & iOS 18.5 Beta 2

**Comment by llvllo** [comment]
> Title: "Settings ⭢ Wi-Fi" stopped working URL: Wi-Fi: prefs:root=WIFI Device: iPhone 13 OS version: 18.2 Language: en

Updated URL: settings-navigation://com.apple.Settings.WiFi
Tested on iPadOS 18.4.1 & iOS 18.5 Beta 2

**Comment by llvllo** [comment]
> prefs:root=Privacy&path=LOCATION

Opens Settings > Privacy & Security > Location Services correctly, Tested on iPadOS 18.4.1 & iOS 18.5 Beta 2
